templates/base/head_script.tmpl

```diff
@@ -44,4 +47,4 @@ If you introduce mistakes in it, Gitea JavaScript code wouldn't run correctly.
 	{{/* in case some pages don't render the pageData, we make sure it is an object to prevent null access */}}
 	window.config.pageData = window.config.pageData || {};
 </script>
-<script src="{{AssetUrlPrefix}}/js/webcomponents.js?v={{AssetVersion}}"></script>
+<script type="module" src="{{AssetUrlPrefix}}/webcomponents.js?v={{AssetVersion}}"></script>
```

Comment:
I guess it causes UI flicker if the web components are loaded after the tags.

Reply:
You mean because of the automatic `defer` that module scripts do?

Reply:
yes

Reply:
I do not see any flicker with timestamps at least, so I assume that part must still be working.

Reply:
How about adding a "time.Sleep" in the template code to simulate a laggy network, or use the browser's debugger tool to simluate a laggy network.

If the page is partially rendered, the custom web component tags won't have its code until the JS file is executed.

Reply:
Slowness can be simulated in browser as well. At least Firefox has an option to simulate slow network.

Reply:
Chrome does show a flicker on the timestamps, while Firefox doesn't. Need to fix this somehow.

Reply:
Done, now not using `type=module` for webcomponents.